cfgov/v1/tests/test_links.py

```diff
@@ -22,6 +26,41 @@ def test_gov_link(self):
         self.assertIn('class="icon-link_text"', output)
 
 
+class LinkTest(TestCase):
+
+    def setUp(self):
+
+        self.page = mock.MagicMock()
+        self.request = mock.MagicMock()
+        self.request.url = 'http://localhost:8000/about-us/blog/'
+        self.page.url = 'http://localhost:8000/about-us/blog/we-want-hear-public-about-payday-loans/'
```

Comment:
if page.url simply appends onto the request url, it might be better to write it using string interpolation, i.e.:

`self.page.url = "%swe-want-hear..." % self.request.url`